common-app-lib/src/cmr/common_app/config.clj

```diff
@@ -8,7 +8,7 @@
 (defconfig collection-umm-version
   "Defines the latest collection umm version accepted by ingest - it's the latest official version.
    This environment variable needs to be manually set when newer UMM version becomes official"
-  {:default "1.17.3"})
```

Comment:
the second this is committed on are "on the clock" to finish this effort. Should this not be the last change?

Reply:
off line we talked about how this risk is low.